src/config.js

```diff
@@ -30,20 +30,17 @@ const config = {
     jwtSecret: requireProcessEnv('JWT_SECRET'),
     mongo: {
       options: {
-        db: {
-          safe: true
-        }
+        useCreateIndex: true,
+        useNewUrlParser: true,
+        useUnifiedTopology: true,
+        debug: true
       }
     }
   },
   test: { },
   development: {
     mongo: {
-      uri: `mongodb+srv://${requireProcessEnv('DB_UNAME')}:${requireProcessEnv('DB_PASS')}@cluster0-qjtrk.mongodb.net/test?retryWrites=true&w=majority`,
-      options: {
-        useNewUrlParser: true,
-        debug: true
-      }
+      uri: 'mongodb://localhost:27017' 
```

Comment:
Seems its necessary to specify the database also (like this: uri: 'mongodb://localhost:27017/dev') otherwise its throwing this error:  throw MongoError.create({ message: 'database name must be a string', driver: true });